package.json

```diff
@@ -23,7 +23,7 @@
     "build-dist-min": "webpack --bail --config node_modules/builder-victory-component/config/webpack/webpack.config.js --colors",
     "build-dist-dev": "webpack --bail --config node_modules/builder-victory-component/config/webpack/webpack.config.dev.js --colors",
     "build-dist": "builder run clean-dist && builder run build-dist-min && builder run build-dist-dev",
-    "build-lib": "builder run clean-lib && babel src -d lib --copy-files",
+    "build-lib": "builder run clean-lib && ('node_modules/builder-victory-component/node_modules/.bin/babel' src -d lib --copy-files || babel src -d lib --copy-files)",
```

Comment:
Also, as a reminder, here's how builder augments `PATH`:

1. `ROOT/node_modules/archetype/node_modules/.bin`
2. `ROOT/node_modules/.bin`
3. Old `PATH`

So , we **already try for** `node_modules/builder-victory-component/node_modules/.bin/babel` first before anywhere else. Let's chat what's going on here because remember that resolution order, this looks like it should be definitely reverted...